core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2404,6 +2406,10 @@ public boolean isDefaultBuildDir() {
         return DEFAULT_BUILDS_DIR.equals(buildsDir);
     }
 
+    private boolean isDefaultWorspaceDir() {
```

Comment:
Typo

Reply:
Worspace -> Workspace in case you cannot see it, took me a couple reads :smile: